person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, age
+
+  def initialize(type, age, _parent_permission, name = 'unkown')
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @type = type
+    @parent_permission = true
+  end
+
+  def can_use_services?
+    @type == age or @type == @parent_permission
+  end
```

Comment:
- [ ] This method should return true if the `of_age` method returns `true` or if `@parent_permisssion` returns true. Kindly reflect it👍